test/src/test/java/hudson/security/HudsonPrivateSecurityRealmFIPSTest.java

```diff
@@ -30,9 +30,12 @@
 import static org.hamcrest.Matchers.not;
 import static org.hamcrest.Matchers.startsWith;
 import static org.junit.Assert.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
```

Comment:
This looks like unnecessary use of mocks, prefer `WebClient`.

Reply:
Fixed in recent commit